addOns/automation/src/test/java/org/zaproxy/addon/automation/jobs/ExitStatusJobUnitTest.java

```diff
@@ -348,22 +349,64 @@ void shouldSetExitCode(String alertrisk, String exitcode) {
         assertThat(ExtensionAutomation.getExitOverride(), is(equalTo(Integer.parseInt(exitcode))));
     }
 
-    private JobResultData getTestData(String alertLevel) {
-        Alert alert = new Alert(-1, JobUtils.parseAlertRisk(alertLevel), 2, "test");
+    @ParameterizedTest
+    @CsvSource({
+        "HIGH,MEDIUM,4",
+        "medium,medium,3",
+        "low,medium,2",
+        "High,False Positive,0",
+        "Medium,False Positive,0",
+        "Low,False Positive,0"
+    })
+    void shouldSetExitCodeExcludingFalsePositive(
```

Comment:
Would be good we tested with more than one alert.

Reply:
To be a practical test (and I'm up for doing this) wouldn't I need to generate a collection of somewhat random Alerts, then check the max risk, and assert the proper exit code?

Reply:
No, it does not need to be random we just need to check that we process other alerts ignoring FPs.